test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -44,32 +43,19 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 	} );
 
 	describe( 'Blocks', function () {
-		let blockHandle: ElementHandle;
-
 		it( 'Enter post title', async function () {
 			await gutenbergEditorPage.enterTitle( title );
 		} );
 
 		it( 'Enter post text', async function () {
 			await gutenbergEditorPage.enterText( quote );
 		} );
-
-		it( 'Add image block', async function () {
-			blockHandle = await gutenbergEditorPage.addBlock(
-				ImageBlock.blockName,
-				ImageBlock.blockEditorSelector
-			);
-		} );
-
-		it( 'Remove image block', async function () {
-			await gutenbergEditorPage.removeBlock( blockHandle );
-		} );
```

Comment:
This is semi-related, in that we already have Media blocks being tested in its own suite, so this felt redundant.